controller/adoptopenebs/reconciler.go

```diff
@@ -426,10 +426,19 @@ func (p *Planner) addAdoptOpenEBSLabels(component *unstructured.Unstructured) er
 // components and their configuration.
 func (p *Planner) getDesiredOpenEBS() error {
 	openebs := &unstructured.Unstructured{}
+	openebsLabels := make(map[string]string)
+	// get the adoption job ID label which tells the ID of the job which has triggered
+	// the adoption process.
+	if p.ObservedAdoptOpenEBS.GetLabels() != nil {
+		if jobID, exist := p.ObservedAdoptOpenEBS.GetLabels()[types.KeyAdoptionJobID]; exist {
+			openebsLabels[types.KeyAdoptionJobID] = jobID
```

Comment:
Is this something like adopted due to some AdoptOpenEBS?
Can we write verbose code comments which makes things very clear?
Or probably a doc.go in this package that explains everything?

Reply:
Why the name JobID? Is it Kubernetes Job?

Reply:
It is something like some external jobs or processes triggering this adoption process and in order to be identified which job or process has triggered this adoption process, a user or tool can make use of this label.